lib/openai/client.rb

```diff
@@ -54,5 +54,32 @@ def transcribe(parameters: {})
     def translate(parameters: {})
       OpenAI::Client.multipart_post(path: "/audio/translations", parameters: parameters)
     end
+
+    def token_count(parameters: {})
```

Comment:
This feels strange to me living inside the `Client` class.  I'd expect public methods in this context to interface with OpenAI in some way.  Consider moving this to a new utility class or included on the top-level `OpenAI` namespace.